### PR TITLE
Disable polling of config parameters and update in a separate loop

### PR DESCRIPTION
**Pull request by GDYendell**
Some tweaks to the FastCS core could make this a bit neater, but it works for now.

### CODE REVIEW

File: src/eiger_fastcs/eiger_controller.py [hunk 1]
```diff
@@ -52,17 +70,16 @@ class LogicHandler:
...
-        self,
-        controller: "EigerController",
-        attr: AttrW,
-        value: Any,
-    ) -> None:
+    async def put(self, _: "EigerController", attr: AttrW, value: Any) -> None:
         await attr.set(value)
 
 
+EIGER_HANDLERS: dict[str, Type[EigerHandler]] = {
```
Comment (TBThomas56) [inline src/eiger_fastcs/eiger_controller.py:77]: what is the need for tow handlers? Is it to separate changing values to fixed?

Comment (GDYendell) [reply src/eiger_fastcs/eiger_controller.py:77]: To allow us to have config parameters that aren't polled periodically. They can be updated when they appear in the response to a put request.

File: src/eiger_fastcs/eiger_controller.py [hunk 2]
```diff
@@ -71,15 +88,21 @@ class EigerController(Controller):
...
+    # Logic Parameters
+    manual_trigger = AttrRW(Bool(), handler=LogicHandler())
+    stale_parameters = AttrR(Bool())
 
     def __init__(self, settings: IPConnectionSettings) -> None:
         super().__init__()
         self._ip_settings = settings
 
+        # Parameter update logic
+        self._stale_parameters: set[str] = set()
```
[the remark this answers is not in the document]
Comment (GDYendell) [reply src/eiger_fastcs/eiger_controller.py:103]: They are parameters that have appeared in the response to a put request and are queued to be updated. This is useful for clients to wait on to be sure that all of its commands have propagated down to the detector and everything is up to date.

**Comment by GDYendell** [comment]
I need to remind myself if there is a good reason that the [current logic](https://github.com/dls-controls/eiger-detector/blob/master/python/src/eiger_detector/control/eiger_detector.py#L687) is complicated and if we should just clear the set and let it fill up again while fetching. I think if the same parameter is added again, we can't be sure if we updated before or after, so we should just update it again.

**Review by TBThomas56 — COMMENTED** [review]
Gary - "Is a good reason that the [current logic](https://github.com/dls-controls/eiger-detector/blob/master/python/src/eiger_detector/control/eiger_detector.py#L687) is complicated and if we should just clear the set and let it fill up again while fetching. I think if the same parameter is added again, we can't be sure if we updated before or after, so we should just update it again."

**Pull request merged**